draft-ietf-cbor-serialization.md

```diff
@@ -580,21 +587,20 @@ It suggests, for example, that all NaNs may be encoded as a half-precision quiet
 This section is distinct from the Core Deterministic Encoding Requirements and represents an optional alternative for handling NaNs.
 
 
-## Divergence from {{-cbor}} {#NaNCompatibility}
+## Divergence from RFC 8949 {#NaNCompatibility}
+
+Orindary and deterministic serialization defined in this document diverge from the preferred serialization requirement in {{-cbor}} for shortest-length encoding of NaNs in ordinary and deterministic serializations.
 
-Orindary and deterministic serialization defined in this document diverge from the preferred serialization requirement in {{-cbor}} for shortest-length encoding of NaNs:
+In ordinary and deterministic serializations, non-trivial NaNs are not allowed.
 
-- Ordinary serialization: Non-trivial NaNs are not allowed.
-  While ordinary serialization largely aligns with preferred serialization, it does not in the case of non-trivial NaNs.
-- Deterministic serialization: Because deterministic serialization inherits from ordinary serialization, it also does not allow non-trivial NaNs.
-  This is the single aspect of deterministic serialization that is different from {{Section 4.2.1 of -cbor}}.
+The treatment of non-trivial NaNs is the only aspect of deterministic serialization that is different from {{Section 4.2.1 of -cbor}}.
```

Comment:
How about this:
The treatment of non-trivial NaNs is the sole difference between deterministic serialization and {{Section 4.2.1 of -cbor}}.